src/components/entity/ha-entity-dropdown.html

```diff
@@ -0,0 +1,67 @@
+<link rel="import" href="../../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../../bower_components/vaadin-combo-box/vaadin-combo-box.html">
+<link rel="import" href="../../../bower_components/paper-listbox/paper-listbox.html">
+<link rel="import" href="../../../bower_components/paper-item/paper-icon-item.html">
+<link rel="import" href="../../../bower_components/paper-item/paper-item-body.html">
+
+<link rel="import" href="./state-badge.html">
+
+<dom-module id="ha-entity-dropdown">
+  <template>
+    <vaadin-combo-box
+      autofocus="[[autofocus]]"
+      label="[[label]]"
+      items='[[computeStates(hass)]]'
+      item-value-path='entity_id'
+      item-label-path='entity_id'
+      value='{{value}}'
+    >
+      <template>
+        <style>
+          paper-icon-item {
+            margin: -13px -16px;
```

Comment:
vaadin-combo-box wraps each item in a `<vaadin-combo-box-item>` with `margin: 13px 16px`. I couldn't find a way around not setting it, so this inverses it.